README.md

```diff
@@ -10,9 +10,9 @@ Note: although the `dp-frontend-renderer` is deprecated there is a transition pe
 
 Other than `dp-renderer` itself, you will need a utility that can combine service-specific and `dp-renderer` assets. We currently use `go-bindata` for this process.
 
-- `dp-renderer`: `go get github.com/ONSdigital/dp-renderer`
+- `dp-renderer`: `go get github.com/ONSdigital/dp-renderer/v2`
 
-> You can specify a version of `dp-renderer` by appending a commit ID or semantic version number to this command. E.g., `go get github.com/ONSdigital/dp-renderer@31d8704`
+> You can specify a version of `dp-renderer` by appending a commit ID or semantic version number to this command. E.g., `go get github.com/ONSdigital/dp-renderer/v2@31d8704`
```

Comment:
I'd use a valid v2 commit so that this command works, although it is unlikely anyone will use it
```suggestion
> You can specify a version of `dp-renderer` by appending a commit ID or semantic version number to this command. E.g., `go get github.com/ONSdigital/dp-renderer/v2@d27f174`
```